Example/PaymentSheet Example/PaymentSheet Example/ExampleCustomCheckoutViewController.swift

```diff
@@ -7,15 +7,62 @@
 //
 
 import Foundation
-import StripePaymentSheet
+//  Note: Do not import Stripe using `@_spi(STP)` in production.
+//  This exposes internal functionality which may cause unexpected behavior if used directly.
+// TODO(porter) Remove STP usage before GA
+@_spi (STP) import StripePaymentSheet
 import UIKit
 
 class ExampleCustomCheckoutViewController: UIViewController {
     @IBOutlet weak var buyButton: UIButton!
     @IBOutlet weak var paymentMethodButton: UIButton!
     @IBOutlet weak var paymentMethodImage: UIImageView!
-    var paymentSheetFlowController: PaymentSheet.FlowController!
-    let backendCheckoutUrl = URL(string: "https://stripe-mobile-payment-sheet.glitch.me/checkout")!  // An example backend endpoint
+
+    @IBOutlet weak var hotDogQuantityLabel: UILabel!
+    @IBOutlet weak var saladQuantityLabel: UILabel!
+    @IBOutlet weak var hotDogStepper: UIStepper!
+    @IBOutlet weak var saladStepper: UIStepper!
+    @IBOutlet weak var subtotalLabel: UILabel!
+    @IBOutlet weak var salesTaxLabel: UILabel!
+    @IBOutlet weak var totalLabel: UILabel!
+    @IBOutlet weak var subscribeSwitch: UISwitch!
+
+    private var paymentSheetFlowController: PaymentSheet.FlowController!
+    private let baseUrl = "https://stripe-mobile-payment-sheet-custom.glitch.me"
```

Comment:
Should we pay for this server so it doesn't sleep? I'm worried since the server goes to sleep it may cause tests to fail.